src/controls/PrimitiveArrayControl.tsx

```diff
@@ -66,80 +71,67 @@ export function PrimitiveArrayControl({
   const options: ArrayControlOptions =
     (uischema.options as ArrayControlOptions) ?? {}
 
-  const rules: Rule[] = [
-    { required: required, message: `${label} is required` },
-  ]
-
-  const style = { marginBottom: "0px" }
-
   return (
-    <Form.Item label={label} required={required}>
-      <Form.List name="names" initialValue={data as unknown[] | undefined}>
-        {(fields, { add, remove }, { errors }) => {
-          fields.length === 0 && add()
-          return (
-            <>
-              <Row justify={"start"}>
-                <Col>
-                  {fields.map((field, index) => (
-                    <Form.Item key={index} rules={rules} style={style}>
```

Comment:
This formitem gets deleted

Reply:
the required rule should be applied by whatever control the dispatch dispatches to